CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+This entire repository is simply a discussion repo. So, the main way to contribute is to jump into the issues and start chatting!
+
+## How to write and comment in issues
+
+If you're opening a new issue, it's better to err on the side of more detail than less. For instance, [this is a good issue](https://github.com/probot/friction/issues/29), which specifies what @bkeepers had issues with, what they wanted, and potentially how to fix it. [This one](https://github.com/probot/friction/issues/5) was less good, because just saying something sucks doesn't actively help contribute to the conversation regarding how to fix it. Like all communication: be clear, be honest, and try and be constructive.
+
+In this repo, we encourage you to say "I also have this problem!" This will help us know a bit more about what we can improve, and what is causing friction for people. Unlike on the rest of GitHub, comments are actually more useful as signals for us than reactions!
+
+## Questions?
+
+Want to chat with Probot users and contributors before posting here? [Join us in Slack](https://probot-slackin.herokuapp.com/)!
+
+If you have meta-questions about this repo, feel free to open an issue about them, too.
+
+## Bots
```

Comment:
Nit: could we avoid `Bots` and use `Apps` everywhere in this section? 😬

Reply:
So, I'm confused about this. Can you point me to the discussion about why `bots` isn't great?

Reply:
@hiimbex I changed bot to apps. Let's move that battle elsewhere. :)